app/src/main/java/me/martinez/sergio/daggertwobasearchitecture/activities/Section.java

```diff
@@ -0,0 +1,10 @@
+package me.martinez.sergio.daggertwobasearchitecture.activities;
+
+public class Section {
```

Comment:
No... that was a step I thought it made sense but it didn't, and forgot to remove the class. (TODO)